src/tests/data/restore.test.ts

```diff
@@ -483,4 +484,55 @@ describe("restore", () => {
     );
     expect(restoredData.appState.name).toBe(stubImportedAppState.name);
   });
+
+  it("bump versions of local duplicate elements when supplied (restore)", () => {
```

Comment:
```suggestion
  it("bump versions of local duplicate elements when supplied", () => {
```
not needed since its under `describe('restore')`